.github/workflows/ci.yml

```diff
@@ -38,3 +37,17 @@ jobs:
         uses: ammaraskar/sphinx-action@master
         with:
           docs-folder: "docs/"
+
+      - name: Lint Dockerfiles
+        uses: jbergstroem/hadolint-gh-action@v1
+        with:
+          dockerfile: "./rest_api_gcbm/Dockerfile
+            ./rest_api_flint.example/Dockerfile
+            ./local/rest_api_flint.example/Dockerfile
+            ./local/rest_api_gcbm/Dockerfile
+            ./layered/cloud-run/Dockerfile
+            ./layered/compute-engine/Dockerfile
+            ./layered/ingress/Dockerfile"
+
+          error_level: 0
+          output_format: tty
```

Comment:
This works well. However, this step is a part of the docs linting job. Can you refactor this to work with the previous job for better compatibility?

Reply:
I have made the changes.